server/api.js

```diff
@@ -6,7 +6,43 @@ const router = Router();
 
 router.get("/", (_, res) => {
 	logger.debug("Welcoming everyone...");
-	res.json({ message: "Hello, world!" });
+	res.json({ message: "WELCOME TO LOVE ME TENDER SITE" });
 });
 
-export default router;
+router.get("/skills", (req, res) => {
+
+	const skills = [
+		"Website",
+		"Android",
+		"iOS",
+		"Backend",
+		"Frontend",
+		"Full-stack",
+	];
+
+	skills.sort();
+
+	res.json({ skills });
+});
+
+router.post("/api/publish-tenders", (req, res) => {
+    const formData = req.body;
+    logger.debug("Received form data:", formData);
+
+    if (!formData.title || formData.title.length < 10 || formData.title.length > 50) {
+        return res.status(400).json({ error: "Tender Title must be between 10 and 50 characters." });
```

Comment:
Don't return after the first error. Validate all data, group all the errors and send them all in the response.

Reply:
See my comment above, I said exactly the same thing for `PublishTenderForm.js`